SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -406,6 +406,57 @@ FSpatialNetGUIDCache::FSpatialNetGUIDCache(USpatialNetDriver* InDriver)
 {
 }
 
+typedef TMap<UObject*, uint32> FSubobjectToOffsetMap;
```

Comment:
minor: prefer using over typedef in accordance with the style guide.

Reply:
I thought we based on Epic's coding standard, which specifies not using `using` in global scope, even if it's in a cpp file: https://docs.unrealengine.com/en-US/ProductionPipelines/DevelopmentSetup/CodingStandard/#namespaces
> * Do not put using declarations in the global scope, even in a .cpp file (it will cause problems with our "unity" build system.)

Reply:
That guideline refers to a different usage of `using`, i.e. for pulling namespaces into global scope. Using it as the functional equivalent of `typedef` is fine and preferred within the GDK code base.

Reply:
OK!